frontend/src/lib/components/TeamInvitation.js

```diff
@@ -12,16 +13,29 @@ export function TeamInvitationLink({ user }) {
 }
 
 export function TeamInvitationContent({ user }) {
-    return (
-        <div>
-            <p>
-                <TeamInvitationLink user={user} />
-            </p>
-            Invite teammates with the link above.
-            <br />
-            Build an even better product, <i>together</i>.
-        </div>
-    )
+    const url = window.location.origin
+    const signup_token_state = user['team']['signup_token']
+    const signup_data = `{'team':{'signup_token': ${signup_token_state}}}`
+    if (signup_token_state != null) {
+        return (
+            <div>
+                <p>
+                    <TeamInvitationLink user={user} />
+                </p>
+                Invite teammates with the link above.
+                <br />
+                Build an even better product, <i>together</i>.
+                <br />
+                Link Sharing: <Switch defaultChecked onChange={() => api.update(url, signup_data)} />
```

Comment:
Actually, since this is updating `user`, it should use `userLogic.actions.userUpdateRequest`. Sorry for the confusion! Kea logics are way better for state though then syncing state and API responses in components.